packages/playwright-core/src/client/fetch.ts

```diff
@@ -336,4 +336,17 @@ function isJsonContentType(headers?: HeadersArray): boolean {
       return value === 'application/json';
   }
   return false;
-}
\ No newline at end of file
+}
+
+function objectToArray(map?:  { [key: string]: any }): NameValue[] | undefined {
```

Comment:
There is `envObjectToArray` that does pretty much the same.

Reply:
But this one is used for fetch params...